app/interfaces/MatchTeamStatsDTO.java

```diff
@@ -0,0 +1,167 @@
+package interfaces;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TeamStatsDTO contains all of the information necessary for the stats array list
+ *
+ * @author jshyo
+ *
+ */
+public class MatchTeamStatsDTO extends MatchTeamDTO{
+    // formatter
+    public static final DecimalFormat DF = new DecimalFormat( "#.##" );
+
+    // Display stats
+    private String opponent;
+    private String result;
+    private Integer pointsTotal;
+    private Integer pointsAgainst;
+    private List<TeamTossupValuesDTO> tossupValues;
+    private Integer TUH;

```

Comment:
We should follow camel-case naming conventions, so this would be 'tuh`, PPB would be 'ppb', etc.